Fix PDN scenario failing

```text
[2022/11/14 05:11:58][INFO] ----------------------------------------------
[2022/11/14 05:11:58][INFO] Initializing logger 2022-11-14 05:11:58.557115
[2022/11/14 05:11:58][INFO] ----------------------------------------------
[2022/11/14 05:11:58][INFO] $ C:\dotnetbuild\work\AD890994\p\startup\Startup.exe --app-exe dotnet --metric-type PDN --trace-name "Startup - PaintDotNet - FDD Publish_startup" --gui-app true --process-will-exit false --iterations 5 --timeout 60 --warmup true --working-dir C:\dotnetbuild\work\AD890994\w\A8310965\e --report-json-path traces\perf-lab-report.json --trace-directory traces --scenario-name "Startup - PaintDotNet - FDD Publish" --app-args pub\paintdotnet.dll --measurement-delay 10 --run-without-exit false --run-with-dotnet
[2022/11/14 05:11:58][INFO] Running dotnet (args: "pub\paintdotnet.dll")
[2022/11/14 05:11:58][INFO] Working Directory: C:\dotnetbuild\work\AD890994\w\A8310965\e
[2022/11/14 05:11:58][INFO] Iteration set up:  (args: )
[2022/11/14 05:11:58][INFO] Iteration clean up:  (args: )
[2022/11/14 05:11:58][INFO] =============== Warm up ================
[2022/11/14 05:11:58][INFO] ***Test***
[2022/11/14 05:12:08][INFO] Process 11052 failed to run. Result: ExitedEarly
[2022/11/14 05:12:08][ERROR] Process exited with status 4294967295
[2022/11/14 05:12:08][INFO] Run failure registered
Traceback (most recent call last):
  File "test.py", line 22, in <module>
    main()
  File "test.py", line 18, in main
    runner.run()
  File "C:\dotnetbuild\work\AD890994\p\shared\runner.py", line 222, in run
    startup.runtests(self.traits)
  File "C:\dotnetbuild\work\AD890994\p\shared\startup.py", line 152, in runtests
    RunCommand(startup_args, verbose=True).run()
  File "C:\dotnetbuild\work\AD890994\p\scripts\performance\common.py", line 251, in run
    returncode, quoted_cmdline)
subprocess.CalledProcessError: Command '$ C:\dotnetbuild\work\AD890994\p\startup\Startup.exe --app-exe dotnet --metric-type PDN --trace-name "Startup - PaintDotNet - FDD Publish_startup" --gui-app true --process-will-exit false --iterations 5 --timeout 60 --warmup true --working-dir C:\dotnetbuild\work\AD890994\w\A8310965\e --report-json-path traces\perf-lab-report.json --trace-directory traces --scenario-name "Startup - PaintDotNet - FDD Publish" --app-args pub\paintdotnet.dll --measurement-delay 10 --run-without-exit false --run-with-dotnet' returned non-zero exit status 4294967295.
2022-11-14T13:12:09.430Z	INFO   	run.py	run(48)	main	Beginning reading of test results.
2022-11-14T13:12:09.430Z	INFO   	run.py	__init__(42)	read_results	Searching 'C:\dotnetbuild\work\AD890994\w\A8310965\e' for test results files
2022-11-14T13:12:09.430Z	INFO   	run.py	__init__(42)	read_results	Searching 'C:\dotnetbuild\work\AD890994\w\A8310965\uploads' for test results files
2022-11-14T13:12:09.430Z	WARNING	run.py	__init__(55)	read_results	No results file found in any of the following formats: xunit, junit, trx
2022-11-14T13:12:09.430Z	INFO   	run.py	packing_test_reporter(30)	report_results	Packing 0 test reports to 'C:\dotnetbuild\work\AD890994\w\A8310965\e\__test_report.json'
2022-11-14T13:12:09.446Z	INFO   	run.py	packing_test_reporter(33)	report_results	Packed 1410 bytes
['Startup - PaintDotNet - FDD Publish' END OF WORK ITEM LOG: Command exited with 1]
```

--- a/src/scenarios/paintdotnet/test.py
+++ b/src/scenarios/paintdotnet/test.py
@@ -5,6 +5,8 @@ from shared import const
 EXENAME = 'paintdotnet'
 
 def main():
+    os.environ['DOTNET_ROLL_FORWARD'] = 'LatestMajor'
+    os.environ['DOTNET_ROLL_FORWARD_TO_PRERELEASE'] = '1'
     traits = TestTraits(exename=EXENAME,
                         guiapp='true',
                         startupmetric='PDN', 

